src/components/expression/expression-renderer.js

```diff
@@ -551,6 +713,9 @@ genotet.ExpressionRenderer.prototype.drawMatrixCells_ = function() {
   var heatmapRects = heatmapRows.selectAll('rect').data(_.identity);
   heatmapRects.enter().append('rect');
   heatmapRects
+    .attr('class', function(value, i, j) {
+      return 'cell row' + j + ' column' + i;
```

Comment:
If every rect is a cell, maybe use "g.heatmap > rect" or similar, and avoid adding cell class to every rect.